src/pages/docs/messages/annotations.mdx

```diff
@@ -0,0 +1,529 @@
+
+The `clientId` specified in the [client options](/docs/api/realtime-sdk#client-options) will be associated with the published delete annotation.
+
+<Aside data-type='note'>
+Some summarization methods require the client to be identified with a `clientId` in order to delete annotations. For more information, see [annotation summaries](#annotation-summaries).
+</Aside>
+
+When publishing a delete annotation, specify the [annotation type](#annotation-types) on the `type` field of the annotation object.
+
+You can also optionally specify a `name` for the annotation which is used by certain [summarization methods](#annotation-summaries) that aggregate by `name`.
+
+<Code>
+```javascript
+// Create an Ably Realtime client specifying the clientId that will
+// be associated with annotations published by this client
+const realtime = new Ably.Realtime({ key: '{{API_KEY}}', clientId: 'my-client-id' });
+
+// Create a channel in a namespace called `annotations`
+// which has message annotations enabled 
+const channel = realtime.channels.get('annotations:example');
+
+// Delete a 'delivered' annotation
+await channel.annotations.delete(message.serial, {
+  type: 'receipts:flag.v1',
+  name: 'delivered'
+});
+```
+
+```nodejs
+// Create an Ably Realtime client specifying the clientId that will
+// be associated with annotations published by this client
+const realtime = new Ably.Realtime({ key: '{{API_KEY}}', clientId: 'my-client-id' });
+
+// Create a channel in a namespace called `annotations`
+// which has message annotations enabled 
+const channel = realtime.channels.get('annotations:example');
+
+// Delete a 'delivered' annotation
+await channel.annotations.delete(message.serial, {
+  type: 'receipts:flag.v1',
+  name: 'delivered'
+});
+```
+</Code>
+
+
+## Subscribing to annotation updates <a id="subscribe" />
+
+There are two ways to subscribe to annotation updates:
+
+1. **Annotation summaries** (recommended) - Receive aggregated summaries of all annotations for a message
+2. **Individual annotations** - Receive individual events when annotations are published or deleted
+
+### Subscribing to annotation summaries <a id="subscribe-annotation-summaries" />
+
+The recommended way to receive annotation updates is through annotation summaries. These events provide a summary of the complete, current state of all annotations for a message whenever annotations are published or deleted.
+
+Annotation summaries are delivered to subscribers as messages with an `action` of `message.summary`. These messages have a `summary` field which provides a summary of all the annotations for the message identified by the `serial` field on the summary message.
+
+The value of the `summary` field is an object where the keys are the [annotation types](#annotation-types). The exact structure of the value of each key depends on the [summarization method](#annotation-summaries) specified in the annotation type.
```

Comment:
`summary` doesn't exist in the table describing the properties of an annotation message.